src/test/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionRoutesV1Spec.scala

```diff
@@ -293,4 +293,33 @@ class GoogleExtensionRoutesV1Spec extends GoogleExtensionRoutesSpecHelper with S
       status shouldEqual StatusCodes.NotFound
     }
   }
+  "POST /api/google/v1/user/signedUrlForBlob" should "200 with a signed url" in {
```

Comment:
If you include the PSA to use, this could (should?) be:
```suggestion
  "POST /api/google/v2/user/petServiceAccount/{project}/signedUrlForBlob" should "200 with a signed url" in {
```

Reply:
I just worry this is going to get confusing

Reply:
I want to get rid of the ability to choose which PSA to use. That might be a "controversial" move, but I think its a better design to just use the Arbitrary PSA

Reply:
I'm not sure from a permission standpoint.  If you're signing a URL from a given project, it seems wrong to me that the service account would be from some random other project (and it makes the call not deterministic, which makes me nervous).  I get having an option do do this arbitrarily but it seems like we should support both...

Reply:
Is there somewhere that we could at least put an explanation of why it's an "arbitrary" PSA? To me that is a code smell. If I came in here to update this code and saw that usage without a comment or doc I'd be concerned about why it's doing that and what implications it might have